redundant-bmc/src/manager.cpp

```diff
@@ -70,26 +80,42 @@ sdbusplus::async::task<> Manager::doHeartBeat()
 
 Role Manager::determineRole()
 {
-    Role role{Role::Unknown};
+    using namespace role_determination;
+
+    RoleInfo roleInfo{Role::Unknown, ErrorCase::noError};
 
     try
     {
+        // Note:  If these returned nullopts, the algorithm wouldn't use
+        //        them anyway because there would be no heartbeat.
+        auto siblingRole = sibling->getRole().value_or(Role::Unknown);
+        auto siblingProvisioned = sibling->getProvisioned().value_or(true);
+        auto siblingPosition = sibling->getPosition().value_or(0xFF);
```

Comment:
getPosition() returns a size_t (as an optional).  The only reason it would return the nulltopt is if the sibling isn't responding, in which case we couldn't get far enough to compare positions anyway.  What do you think i should do here?

Reply:
Was just hoping the 0xFF could be replaced with Role::Unknown

Reply:
Ahh, I see, there is no enum for Position. Oh well.